### PR TITLE
Adds apparent age descriptors

### PR DESCRIPTION
**Pull request by BobJoga**
## About The Pull Request

Title. It's not based on your actual age, therefore *apparent age*.

Can go from very young to elderly, but i made sure the text isn't creepy.

## Why It's Good For The Game

i want to set myself to young so i can make jokes about being a soundcloud rapper

## Changelog
:cl:
add: Added apparent age descriptors to the character setup.
tweak: Generalized descriptors. Now they don't mention species by default, and are therefore respecting human averages.
/:cl:

### CODE REVIEW

File: modular_skyrat/code/datums/descriptors/descriptor_types/apparent_age.dm [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+		"adult",
+		"middle-aged",
+		"elderly",
+		)
+	comparative_value_descriptor_equivalent = "around the same age as you"
+	comparative_value_descriptors_smaller = list(
+		"slightly younger than you",
+		"younger than you",
+		"much younger than you",
+		"juvenile when compared to you",
```
Comment (kappa-sama) [inline modular_skyrat/code/datums/descriptors/descriptor_types/apparent_age.dm:16]: haha god i'm so glad skyrat exists to draw these people away from citadel

**Comment by ForrestWick** [comment]
I'm in hard disagreement to this. This doesn't help with RP, it doesn't encourage RP, is unneeded bloat as far as features go, and seems highly abusable. Flavortext already exists, and is a perfectly fine alternative to stuff like the Bay descriptors, which that PR already has several issues that as far as I'm aware are not fixed.

File: modular_skyrat/code/datums/descriptors/descriptor_types/apparent_age.dm (hunk 1 above)
Comment (BobJoga) [reply modular_skyrat/code/datums/descriptors/descriptor_types/apparent_age.dm:16]: ebin

**Comment by BobJoga** [comment]
> I'm in hard disagreement to this. This doesn't help with RP, it doesn't encourage RP, is unneeded bloat as far as features go, and seems highly abusable. Flavortext already exists, and is a perfectly fine alternative to stuff like the Bay descriptors, which that PR already has several issues that as far as I'm aware are not fixed.

The issues are fixed and i've stated my opinion repeatedly on the discord. I have nothing to add here.

**Comment by calibraptor** [comment]
I'd honestly really rather not. Having age as a descriptor is a slippery slope we do /not/ need to walk the edge of.

File: modular_skyrat/code/datums/descriptors/descriptor_types/apparent_age.dm [hunk 2]
```diff
@@ -0,0 +1,23 @@
+/datum/mob_descriptor/apparent_age
+	name = "apparent age"
+	replace_are = "appear to be"
+	standalone_value_descriptors = list(
+		"very young",
+		"young",
+		"adult",
+		"middle-aged",
+		"elderly",
```
Comment (jjpark-kb) [inline modular_skyrat/code/datums/descriptors/descriptor_types/apparent_age.dm:9]:
```suggestion
		"early adult",
		"middle-aged adult",
		"older adult",
		"elderly",
		"ancient",
```

File: modular_skyrat/code/datums/descriptors/descriptor_types/apparent_age.dm (hunk 1 above)
Comment (jjpark-kb) [inline modular_skyrat/code/datums/descriptors/descriptor_types/apparent_age.dm:16]:
```suggestion
		"barely younger than you",
		"slightly younger than you",
		"younger than you",
		"much younger than you",
```

File: modular_skyrat/code/datums/descriptors/descriptor_types/apparent_age.dm [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+		"slightly younger than you",
+		"younger than you",
+		"much younger than you",
+		"juvenile when compared to you",
+		)
+	comparative_value_descriptors_larger = list(
+		"slightly older than you",
+		"older than you",
+		"much older than you",
+		"dodging a coffin when compared to you",
```
Comment (jjpark-kb) [inline modular_skyrat/code/datums/descriptors/descriptor_types/apparent_age.dm:22]:
```suggestion
		"barely older than you",
		"slightly older than you",
		"older than you",
		"much older than you",
```

**Comment by BobJoga** [comment]
Will fix this tomorrow, thanks @jjpark-kb

**Comment by hugmeorelse** [comment]
No. Why do we need this?

**Pull request closed without merging**